### DISCUSSION TITLE
[Bug]: I'm very confused why I'm unable to login on the app

### DISCUSSION THREAD

**Discussion by LadyHaywood** [advplyr/audiobookshelf-app#1354] [Q&A]
### What was the Problem?

I wanted to try out both the official app but everytime I put in the same address that's on my windows pc it spits out a failed to connet error and I don't know what the problem is. The site doc doesn't really go in-depth about how to set up the adroid app.

I don't know what setting in the windows app or on my computer I need to change. The address it used is localhost:13378 but I've also read that it could also be 127.0.0.1, I tried to just log in with my own ip adress to see if that would work and nothing connets the app to the library.

### Steps to Reproduce the Issue

1. Put localhost address into server address field

### What was Expected?

To be logged in

### Phone Model

Samsung S22 Ultra

### Phone OS

13

### Audiobookshelf App Version

Android App - 0.9.76

### Installation Source

Google Play Store

### Additional Notes

On another note with the official app itself I thought the "local media" folder set up was going to read the audiobook files on my phone but it didn't so I'm confused about that as well.

**Comment by nichwall** [comment]
`localhost` and `127.0.0.1` are fancy computer words for "this device", which is why it works on your computer and not on your phone (a different device). You will need to use the local IP of your computer/server, which will likely be something like `192.168.x.x` but may be different based on your specific router or network settings.

The apps are designed to specifically work only with media you have downloaded from the server within the app. The "local media" folder is in case you want to use other apps to listen to the media instead of the ABS app. The ABS app is not designed to work with other media files and this functionality is not planned to be added back in (removed in 0.9.67, almost a year ago).

**Comment by LadyHaywood** [comment]
I did try to use my own IP but it gives me this. I'm on the same wifi but I also still tried to use the address it said my phone was connecting from and it didn't even try to connect. 
![Screenshot_20241026_031406_Audiobookshelf](https://github.com/user-attachments/assets/f54e7b6d-c535-4aca-8619-de7a8a2713ae)

**Comment by nichwall** [comment]
I am assuming you do not have SSL set up on your server (ABS does not provide SSL, so you'll need to set a reverse proxy or similar to use https). Does it work if you enter `http://` instead of `https://`?

**Comment by nichwall** [comment]
It could also be a firewall on your computer preventing other devices from accessing it. Can you access the ABS server using your mobile browser?

**Comment by LadyHaywood** [comment]
I have tried both http and https and it wouldn't connect. I've tried putting using my phone's browser and that wouldn't connect. I don't know what SSL is or how to set that up nor do I know how to check if my firewall is blocking the connection.

**Comment by romainiac** [comment]
had the same issue. just remove the http(s):// entirely , just put in the ip:port
